docs/examples/tests/test_secrets.py

```diff
@@ -93,7 +93,7 @@ def config_entry(monkeypatch, base_uri):
                 return_value=configs.RuntimeConfiguration(
                     config_name="mocked",
                     runtime_name=configs.RuntimeName.QE_REMOTE,
-                    runtime_options={"uri": base_uri, "token": "mocked_token"},
+                    runtime_options={"_uri": base_uri, "_token": "mocked_token"},
```

Comment:
I know this PR is still WIP but I'm letting you know ahead of time.

Are you sure this change is needed? `RuntimeConfiguration` is a model that tightly corresponds to the `config.json` file contents. This change is likely to be incompatible with the state users already have.

Reply:
FYI: adding underscores to these fields is probably the way to go in every other place; here (in the model) I'd advocate for keeping it as-is.